api.yaml

```diff
@@ -1890,7 +1890,7 @@ components:
     ProjectAccess:
       type: object
       properties:
-        memeber_id:
```

Comment:
you don't need to edit api.yml, it will be updated automatically

Reply:
@1ivliev take a note. I merge it, not critical